MyConfig.h

```diff
@@ -48,6 +48,19 @@
 //#define MY_DEBUG
 
 /**
+ * @def MY_DEBUG_OTA
+ * @brief Define MY_DEBUG_OTA to redirect debug prints to given node ID
+ *
+ * This option redirects serial debugging to given node ID, while serial Debugging is disabled.
```

Comment:
I find the line a bit confusing. "while serial  debugging is disabled". You mean to say that when this flag is set, you get no serial debug, instead all debug goes OTA to specified node id, right?

Reply:
Right.